transcom/mymove
# INT B-18836 send email when office account rejected

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18836)

## Summary

When requesting office users are asking for accounts, we want to notify them when they are rejected. This PR covers the notification process of sending an email to that user that contains the rejection reason.

[this article](https://dp3.atlassian.net/wiki/spaces/MT/pages/1467973894/How+to+test+sending+email+locally) explains more about the approach used when testing emails locally

### How to test

1. Start up your client normally, but server with 
`EMAIL_BACKEND=ses AWS_REGION=us-gov-west-1  aws-vault exec transcom-gov-dev -- make server_run`
2. Login as an admin
3. Find a requesting office user (if ya don't have one, change the `status` value in the `office_users` table with `REQUESTED
4. Ensure that the email of that office user is one that you have access to
5. Fill out the rejection reason
6. Send 'em packin' & click the reject button
7. You should now get an email that matches the screenshot that contains the rejection reason you filled out

## Screenshots
![Screenshot 2024-03-27 at 1 17 28 PM](https://github.com/transcom/mymove/assets/136510600/074187a4-ab3a-4419-90a1-2990dcaa1613)
